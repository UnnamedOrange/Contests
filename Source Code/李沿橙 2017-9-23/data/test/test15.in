5
50
744 222 294 716 701 690 349 450 669 152 502 27 576 795 328 228 286 507 550 801 590 163 516 933 376 331 538 865 655 623 329 277 191 243 134 456 73 799 764 364 421 138 842 882 513 511 636 355 860 51 
32
924 638 121 339 181 869 731 444 681 975 544 847 176 354 597 186 783 797 328 866 431 754 552 480 503 634 660 372 208 376 760 485 
45
285 694 957 900 786 282 837 499 883 816 512 104 652 234 591 978 401 573 992 29 15 219 803 476 923 674 116 80 549 799 596 79 438 855 833 742 545 440 43 342 771 126 331 240 844 
49
639 894 120 872 55 479 218 75 367 487 386 250 54 262 261 685 975 681 352 274 224 780 336 148 926 324 734 458 274 339 616 997 767 586 170 422 772 428 666 600 541 538 834 266 350 815 511 499 885 
46
780 211 8 930 151 814 776 607 698 327 977 343 47 676 620 101 190 620 734 158 409 860 322 144 194 229 758 207 957 638 982 958 487 848 239 298 9 202 758 634 932 194 194 991 244 50 
