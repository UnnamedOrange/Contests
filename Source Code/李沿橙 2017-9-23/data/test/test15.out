27 51 134 138 163 191 222 243 277 286 328 329 331 349 355 364 421 450 502 507 511 513 516 550 576 623 636 655 690 701 716 764 795 799 842 860 865 933 882 801 744 669 590 538 456 376 294 228 152 73 
121 176 181 186 328 354 372 376 431 444 485 503 544 552 634 638 660 681 754 760 783 797 847 866 924 975 869 731 597 480 339 208 
15 29 43 79 80 104 126 234 240 282 331 401 438 476 499 512 549 573 591 596 674 694 771 786 799 803 816 833 837 855 883 900 957 978 992 923 844 742 652 545 440 342 285 219 116 
54 55 75 148 218 224 261 262 266 274 274 336 339 350 352 367 422 428 479 487 499 511 541 586 600 639 666 681 734 772 780 815 872 885 926 997 975 894 834 767 685 616 538 458 386 324 250 170 120 
8 9 47 50 101 144 151 158 190 194 194 194 207 211 229 239 244 298 322 327 409 607 620 620 634 638 698 734 758 758 776 780 814 848 930 932 957 958 977 982 991 860 676 487 343 202 
